apps/test/unit/lib/kits/maker/boards/makeStubBoard.js

```diff
@@ -17,12 +17,23 @@ export class MicrobitStubBoard {
     this.updateListeners = [];
     this.analogChannel = new Array(16).fill(0);
     this.myPort = null;
+    this.boardClient_ = null;
+  }
+
+  boardClient_() {
+    this.boardClient_ = {};
```

Comment:
Curious, why do we need this function? Can we assign `this.boardClient = {}` in the constructor?

Reply:
I tried this at first, but the tests failed. When I printed the `stubBoardController` after assigning `this.boardClient_ = {}` in the constructor, this was the output:
```
LOG: MicrobitStubBoard{connect: connect, pinMode: pinMode, setPinMode: setPinMode, digitalWrite: digitalWrite, digitalRead: digitalRead, analogWrite: analogWrite, analogRead: analogRead, displayShow: displayShow, displayPlot: displayPlot, displayClear: displayClear, scrollString: scrollString, scrollInteger: scrollInteger, receivedEvent: receivedEvent, receivedAnalogUpdate: receivedAnalogUpdate, trackDigitalPin: trackDigitalPin, addFirmataEventListener: addFirmataEventListener, addFirmataUpdateListener: addFirmataUpdateListener, streamAnalogChannel: streamAnalogChannel, stopStreamingAnalogChannel: stopStreamingAnalogChannel, setDigitalOutput: setDigitalOutput, getBoardClientWithoutPort: getBoardClientWithoutPort, disconnect: disconnect, reset: reset, trackDigitalComponent: trackDigitalComponent, enableLightSensor: enableLightSensor, connectBoard: connectBoard, clearChannelData: clearChannelData}
```
There is no `boardClient_` key. So this is why I added the function. Do you have any ideas why this is so?

Reply:
Hmm, interesting...I'm not sure why it wouldn't be defined if not in the constructor. I'm okay to leave it as is for now then and we can take a look offline